patches/minecraft/net/minecraft/advancements/Advancement.java.patch

```diff
@@ -1,5 +1,16 @@
 --- a/net/minecraft/advancements/Advancement.java
 +++ b/net/minecraft/advancements/Advancement.java
+@@ -238,6 +_,10 @@
+          return advancement;
+       }
+ 
++      public Advancement save(Consumer<Advancement> consumer, ResourceLocation id, net.minecraftforge.common.data.ExistingFileHelper fileHelper) {
```

Comment:
Use an extension interface. Instead of exposing a patch and class as public API.